frontend/pages/club/[club]/index.js

```diff
@@ -117,18 +120,18 @@ const Club = ({
             subscriptions={subscriptions}
             updateSubscriptions={updateSubscriptions}
           />
-          <Card bordered style={{ marginBottom: '1rem', background: WHITE }}>
+          <StyledCard bordered>
             <StrongText>Basic Info</StrongText>
             <InfoBox club={club} />
             <br />
             <StrongText>Contact</StrongText>
             <SocialIcons club={club} />
-          </Card>
+          </StyledCard>
           {club.how_to_get_involved ? (
-            <Card bordered style={{ marginBottom: '1rem' }}>
+            <StyledCard bordered style={{ paddingBottom: '0.01rem' }}>
```

Comment:
Why have this very small padding bottom?

Reply:
that was me. i forgot why it was acting weird, will take a look once out of class